substrate/frame/benchmarking/Cargo.toml

```
@@ -38,6 +38,9 @@ static_assertions = { workspace = true, default-features = true }
 array-bytes = { workspace = true, default-features = true }
 rusty-fork = { workspace = true }
 sp-keystore = { workspace = true, default-features = true }
+sc-client-db.workspace = true
+sp-state-machine.workspace = true
+sp-externalities.workspace = true
```

Can we stay consistent? 🙈 

```suggestion
sc-client-db = { workspace = true }
sp-state-machine = { workspace = true }
sp-externalities = { workspace = true }
```

---

I actually like this syntax, because it is shorter :see_no_evil:

---

>Can we stay consistent?

We could add a CI check to enforce either. I think the only difference is when adding features, you need to change more. But it is less to type initially 🤷